packages/sourcemap-upload/package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "@replayio/sourcemap-upload",
-			"version": "1.0.6",
+			"version": "1.0.7",
 			"license": "BSD-3-Clause",
```

Comment:
Is all this from an unrelated PR? Can we land that first or in a separate commit so it's easier to see what's changing I this PR?

Reply:
I think this was all from me installing `node-fetch`, but maybe I ran some unnecessary commands alongside that. I can try and reduce the noise here.

Reply:
Should be significantly reduced now!